pipeline_steps/ssh_slave.groovy

```diff
@@ -41,27 +58,31 @@ def destroy(){
   common.conditionalStep(
     step_name: 'Destroy Slave',
     step: {
-      withCredentials([
-        usernamePassword(
-          credentialsId: "service_account_jenkins_api_creds",
-          usernameVariable: "JENKINS_USERNAME",
-          passwordVariable: "JENKINS_API_KEY"
-        )
-      ]){
-        dir("rpc-gating/scripts"){
-          retry(5) {
-            sh """
-              . ${env.WORKSPACE}/.venv/bin/activate
-              pip install 'pip==9.0.1'
-              pip install -c ../constraints.txt jenkinsapi
-              python jenkins_node.py \
-                delete --name "${instance_name}"
-            """
+      node('CentOS'){
```

Comment:
I ran a test job and it failed because when it allocated the new node, it didn't use the same workspace, so the new workspace didn't have the correct inventory.

https://rpc.jenkins.cit.rackspace.net/job/Single-Use-Slave-Example/85/console